alumnos/ssv/Clase 2/ejercicio6.js

```diff
@@ -0,0 +1,32 @@
+const removeAccents = (str) => {
+  return str.normalize("NFD").replace(/[\u0300-\u036f]/g, "");
+};
+
+const numbersHater = (weekDay, number) => {
+  let answer;
+  const userWeekDay = removeAccents(weekDay).toLowerCase();
+  const conditions = [
+    { day: "lunes", hate: number === 12 },
+    { day: "martes", hate: number > 95 },
+    { day: "miercoles", hate: number === 34 },
+    { day: "jueves", hate: number === 0 },
+    { day: "viernes", hate: number % 2 === 0 },
+    { day: "sabado", hate: number === 56 },
+    { day: "domingo", hate: number === 666 || number === -666 },
```

Comment:
Este era un caso muy claro de switch 😅  No digo que tu solución no sea buena, pero si es verdad que un ciclo for siempre será menos óptima que usar condiciones como switch o incluso if anidados 😅

Aun así, muy guay que hayas tirado por objetos 😄